src/commands/install/handle_install_command.py

```diff
@@ -0,0 +1,38 @@
+from logging import getLogger
+from os import getcwd
+from typing import Any
+
+from colorama import Fore
+
+from src.commands.install.extract_info_from_repo_id import extract_info_from_repo_id
+from src.commands.install.install_package_from_repo import install_package_from_repo
+from src.commands.install.pull_package_submodules import pull_package_submodules
+
+
+def handle_install_command(args: Any) -> None:
+    if args.command != "install":
+        return
+
+    logger = getLogger()
+
+    if args.package is not None and args.package != "":
+        package_info = extract_info_from_repo_id(args.package)
+        install_package_from_repo(
+            package_info.name if args.name is None else args.name,
+            package_info.url,
+            repo_root_dir=getcwd(),
+            destination="lib",
+            tag=package_info.version,
+        )
```

Comment:
> can't we just provide package_info here and optional alias? would simplify the interface.
ok

> Also there is no point in passing repo_root_dir if you just pass getcwd there, do you utilize it somewhere else?
Unit tests use `repo_root_dir` different from `cwd`.

Reply:
Understood, so then maybe provide `getcwd()` as a default value of repo_root_dir and make it optional? WDYT?

Reply:
Then I'd have to add another test case and mock `getcwd`.

I think the signature of that function isn't super import because the function is used in one place, and it's not meant to be part of “public API”.